packages/shared/lib/core/governance/utils/isVotingForProposal.ts

```diff
@@ -0,0 +1,25 @@
+import { get } from 'svelte/store'
+
+import { TrackedParticipationOverview } from '@iota/wallet'
+
+import { getParticipationOverview } from '@core/account/api'
+import { selectedAccount } from '@core/account/stores'
+
+import { selectedProposal } from '../stores'
+
+export async function isVotingForProposal(): Promise<boolean> {
+    const overview = await getParticipationOverview(get(selectedAccount)?.index ?? 0)
+    if (overview) {
+        const proposalId = get(selectedProposal)?.id
+        if (proposalId in overview.participations) {
```

Comment:
```suggestion
        if (overview.participations.includes(proposalId) {
```

This is not correct, because it checks if proposalId is part of the prototype chain or object. The following example shows why this can be problematic.
![image](https://user-images.githubusercontent.com/45079109/208084946-6f8ce874-651e-41c3-bb28-308838b79dc7.png)